OctopusDSC/Tests/OctopusDSCHelpers.Tests.ps1

```diff
@@ -131,4 +134,21 @@ Describe "Invoke-OctopusServerCommand" {
             ((Get-MaskedOutput $dbargs) -match "\*\*\*\*").Count -gt 0 | Should Be $true
         }
     }
-}
\ No newline at end of file
+}
+
+Describe "Test-ValidJson" {
+    It "Returns false for known bad json" {
+        Test-ValidJson (Get-Content "$SamplePath\octopus.server.exe-output-when-json-has-exception-prepended.json" -raw) | Should Be $false
+     }
+
+    It "returns true for known good json" {
+        Test-ValidJson (Get-Content "$SamplePath\octopus.server.exe-output-clean.json" -raw) | Should Be $true
+    }
+}
+
+Describe "Get-CleanedJson" {
+    It "Correctly cleans our expected exception-prepended output" {
+        $clean = Get-CleanedJson (Get-Content "$SamplePath\octopus.server.exe-output-when-json-has-exception-prepended.json" -raw)
+        Test-ValidJson $clean | Should Be $true
```

Comment:
Can you also compare the cleaned json to `octopus.server.exe-output-clean.json`?

Reply:
Probably a damn good plan. I'll add that